src/llscan.h

```diff
@@ -8,19 +8,27 @@
 
 namespace llnode {
 
+class LLScan;
+
 typedef std::vector<uint64_t> ReferencesVector;
 
 typedef std::map<uint64_t, ReferencesVector*> ReferencesByValueMap;
 typedef std::map<std::string, ReferencesVector*> ReferencesByPropertyMap;
 typedef std::map<std::string, ReferencesVector*> ReferencesByStringMap;
 
 
+enum ShowArrayLength { SHOW_ARRAY_LENGTH, DONT_SHOW_ARRAY_LENGTH };
```

Comment:
Questions:
* Should this be inside `FindJsObjectVisitor::MapCacheEntry` or it's ok to keep it global?
* There's another `enum` in this file, and the enumerators' names use this format: `kCamelCasedName`. Should I use the same format (kShowArrayLength, kDontShowArrayLength)?

Reply:
@mmarchini I would suggest to put it inside `FindJsObjectVisitor::MapCacheEntry` or `FindJsObjectVisitor`, and yes it should be named in the camel case style.